esi_leap/api/controllers/v1/lease.py

```diff
@@ -31,6 +31,7 @@
 from esi_leap.resource_objects import resource_object_factory as ro_factory
 
 CONF = esi_leap.conf.CONF
+TIME_FILTER = 'within'
```

Comment:
Since this is used in multiple places - here, the tests, the database API - if you want to use a constant, then it's probably better to have a constants.py in esi_leap/common and declare this value there.

Reply:
Okay, I will create constants.py and add it there!